src/annotator/anchoring/pdf.js

```diff
@@ -430,7 +428,6 @@ export function anchor(root, selectors) {
  * @param {Object} options -
  *   Options passed to `TextQuoteAnchor` and `TextPositionAnchor`'s
  *   `toSelector` methods.
- * @return {Promise<[TextPositionSelector, TextQuoteSelector]>}
```

Comment:
The reason TypeScript complained about the return type here is because the inferred types of the result of the `toSelector` method are not what you expect. If you fix the `toSelector` return types then no changes are needed in this module, other than removing the unused `options` parameter.